hack/e2e/eksctl.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+set -uo pipefail
```

Comment:
`set -e` at top since you are explicitly disabling it down below?

Reply:
yeah will do I think using set -e  by default and then disabling it as necessary is better than the reverse.